src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -1143,25 +1143,14 @@ describe('ReactUpdates', () => {
     expect(mounts).toBe(1);
   });
 
-  it('mounts and unmounts are sync even in a batch', () => {
-    var container1 = document.createElement('div');
-    var container2 = document.createElement('div');
-
-    let called = false;
-    class Foo extends React.Component {
-      componentDidMount() {
-        called = true;
-        ReactDOM.render(<div>Hello</div>, container2);
-        expect(container2.textContent).toEqual('Hello');
-        ReactDOM.unmountComponentAtNode(container2);
-        expect(container2.textContent).toEqual('');
-      }
```

Comment:
Isn't this the equivalent test? https://github.com/facebook/react/pull/8661/files/ad7bcdf999ffc2cd891aef00174f245bcfb497c1#diff-4f1eede220a643b4c75172fcaf8daf59R370